### PR TITLE
sortOn

### PR DESCRIPTION
**Pull request by matrix3d**

### CODE REVIEW

**Comment by matrix3d** [comment]
float number

File: frameworks/projects/Core/src/main/flex/org/apache/flex/utils/Language.as [hunk 1]
```diff
@@ -361,5 +370,56 @@ package org.apache.flex.utils
             return boundMethod;
         };
 
+		/**
+		 * @author lizhi
+		 * @param	arr
+		 * @param	names
+		 * @param	opt
+		 */
+		public static function sortOn(arr:Array,names:Object,opt:int=0):void{
```
Comment (adufilie) [inline frameworks/projects/Core/src/main/flex/org/apache/flex/utils/Language.as]: Sort options can also be an array if names is an array http://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/Array.html#sortOn()

Comment (matrix3d) [inline frameworks/projects/Core/src/main/flex/org/apache/flex/utils/Language.as]: yes ,array work now.

Comment (matrix3d) [inline frameworks/projects/Core/src/main/flex/org/apache/flex/utils/Language.as]: https://github.com/matrix3d/spriteflexjs/blob/master/test/src/TestSortOn.as#L23

Comment (matrix3d) [inline frameworks/projects/Core/src/main/flex/org/apache/flex/utils/Language.as]: https://github.com/apache/flex-asjs/pull/6

Comment (matrix3d) [inline frameworks/projects/Core/src/main/flex/org/apache/flex/utils/Language.as]: the adobe flash player array.sorton ,opt array.
just use the opt[0],the other [1] [2],not work

**Pull request merged**